Rx.playground/Pages/Introduction.xcplaygroundpage/Contents.swift

```diff
@@ -1,171 +1,114 @@
-//: [<< Index](@previous)
+//: [Previous](@previous)
 
 import RxSwift
 import Foundation
 
+
 /*:
-# Introduction
 
-## Why use RxSwift?
+## Introduction
+
+### Why use RxSwift?
 
 A vast majority of the code we write revolves around responding to external actions. When a user manipulates a control, we need to write an @IBAction to respond to that. We need to observe Notifications to detect when the keyboard changes position. We must provide blocks to execute when URL Sessions respond with data. And we use KVO to detect changes in variables.
+
 All of these various systems makes our code needlessly complex. Wouldn't it be better if there was one consistent system that handled all of our call/response code? Rx is such a system.
 
-## Observables
-The key to understanding RxSwift is in understanding the notion of Observables. Creating them, manipulating them, and subscribing to them in order to react to changes.
+### Basis
 
-## Creating and Subscribing to Observables
-The first step in understanding this library is in understanding how to create Observables. There are a number of functions available to make Observables.
-Creating an Observable is one thing, but if nothing subscribes to the observable, then nothing will come of it so both are explained simultaneously.
-*/
+The key to understanding RxSwift is in understanding the notion of Observables. Creating them, manipulating them, and subscribing to them in order to react to changes.
 
-/*:
-### empty
-`empty` creates an empty sequence. The only message it sends is the `.Completed` message.
-*/
+The first step in understanding this library is in understanding how to create Observables. There are a number of [functions available to make Observables](Creating_Observables).
```

Comment:
Maybe also use ` ... create observable sequences.` It's really important that people think in terms of sequences.

I think this other part should probably be:
```
There is a number of [factory functions available](Creating_Observables).
```